local/compose.go

```diff
@@ -175,40 +176,151 @@ func (s *composeService) Up(ctx context.Context, project *types.Project, detach
 	if err != nil {
 		return err
 	}
-	for k, network := range project.Networks {
-		if !network.External.External && network.Name == k {
-			network.Name = fmt.Sprintf("%s_%s", project.Name, k)
-			project.Networks[k] = network
-		}
-		network.Labels = network.Labels.Add(networkLabel, k)
-		network.Labels = network.Labels.Add(projectLabel, project.Name)
-		network.Labels = network.Labels.Add(versionLabel, ComposeVersion)
-		err := s.ensureNetwork(ctx, network)
-		if err != nil {
-			return err
+
+	_, err = progress.Run(ctx, func(ctx context.Context) (string, error) {
```

Comment:
Having `progress.Run` in the backend means that the gRPC side has no way to know what is going on when doing `compose up`, ideally we would pass an `io.Writer` if we want to attach

Reply:
ack.
Unfortunately this means we can't implement `Up attached` as a single func, as we need progressWriter to complete before we can pipe logs to the console